charts/alfresco-search-service/values.yaml

```diff
@@ -12,7 +12,6 @@ nodeSelector: {}
 # alfrescoRegistryPullSecrets has to be set
 # More information can be found on
 # https://github.com/Alfresco/acs-deployment/blob/master/docs/helm/registry-authentication.md
-type: search-services
```

Comment:
what are we trying to achieve here?

L5 document this field

Reply:
From what I could see this is not used anywhere

Reply:
I thought it was unused and missed it actually still was